src/proto/grpc/testing/control.proto

```diff
@@ -220,17 +220,17 @@ message Scenarios {
 // once the scenario has finished.
 message ScenarioResultSummary
 {
-  // Total number of operations per second over all clients.
+  // Total number of queries per second over all clients.
   double qps = 1;
-  // QPS per one server core.
+  // QPS per server core.
   double qps_per_server_core = 2;
-  // server load based on system_time (0.85 => 85%)
+  // The percentage of server cpu load based on system time used by the server process (85 => 85%)
   double server_system_time = 3;
-  // server load based on user_time (0.85 => 85%)
+  // The percentage of server cpu load based on user time used by the server process (85 => 85%)
   double server_user_time = 4;
-  // client load based on system_time (0.85 => 85%)
+  // The percentage of client cpu load based on system time used by the client process (85 => 85%)
```

Comment:
saying "by the client process" is not accurate because there can be multiple client processes (in most of our benchmark there's 2 client actually).  Can we just say mention the symbolic formula?